### ISSUE TITLE
Query: Notifications in-app vs push

### ISSUE THREAD

**Issue by samkirkpatrick** [devRant/devRant#68]
In the Settings > Notifications section, we can toggle what we get notified about. Does this affect entries in the "Notifs" feed or just push notifications? It's just that I have some turned off, but they still appear in the Notifs feed.

Would it be possible to align these so that the settings control what appears in both places? Many thanks...

**Comment by cozyplanes** [comment]
Only system notifications are affected by settings tweak, not in app ones

**Comment by filthycoding** [comment]
Basically, you want an extra filter on which notifications should be shown. Something I wanted to request, too

**Comment by samkirkpatrick** [comment]
Yes. I figured the settings were actually only affecting the device push notifications, but I would like to be able to control the contents of the "Notifs" feed, too, and in my head it makes sense to use the same settings.
